src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -696,23 +714,77 @@ cbuf_create(spdid_t spdid, unsigned long size, int cbid)
 	ret = (int)id;
 done:
 	tracking_end(NULL, CBUF_CRT);
-	CBUF_RELEASE();
-
 	return ret;
 free:
 	cmap_del(&cbufs, id);
 	free(cbi);
 	goto done;
 }
 
+int
+cbuf_create(spdid_t spdid, unsigned long size, int cbid)
+{
+	int ret = 0;
+	unsigned int id = (unsigned int)cbid;
+
+	printl("cbuf_create\n");
+	CBUF_TAKE();
+	ret = __cbuf_create(spdid, size, cbid, 0);
+	CBUF_RELEASE();
+
+	return ret;
+}
+
+int
+__map_and_meta(spdid_t spdid, int cbid, int size, vaddr_t daddr)
+{
+	struct cbuf_comp_info *cci;
+	struct cbuf_info *cbi;
+	struct cbuf_maps *map;
+	vaddr_t dest;
+	void *page;
+	int ret = -EINVAL;
+
+	cci        = cbuf_comp_info_get(spdid);
+	if (!cci) {printd("no cci\n"); goto done; }
+	cbi        = cmap_lookup(&cbufs, cbid);
+	if (!cbi) {printd("no cbi\n"); goto done; }
+	/* shouldn't cbuf2buf your own buffer! */
+	if (cbi->owner.spdid == spdid) {printd("owner\n"); goto done;}
+
+	map        = malloc(sizeof(struct cbuf_maps));
+	if (!map) {printd("no map\n"); ERR_THROW(-ENOMEM, done); }
+	if (size > cbi->size) {printd("too big\n"); goto done; }
+	assert((int)round_to_page(cbi->size) == cbi->size);
+	size       = cbi->size;
+	
+	if (daddr) {
+		dest = daddr;
+	}
```

Comment:
Ah yes. This doesn't adhere to the style guide. Most likely there was more code there that got deleted, like a print.